packages/@atjson/offset-annotations/src/annotations/group.ts

```diff
@@ -17,7 +17,13 @@ import { BlockAnnotation } from "@atjson/document";
  * across multiple documents.
  */
 export class Group<T = {}> extends BlockAnnotation<
-  T & { artDirection: string }
+  T & {
```

Comment:
If we want to treat carousels as an art-direction of a group, we should probably have the `anchorName` on the `group-item` as well so that the carousel can scroll a specific item into view.